app/uk/gov/hmrc/vatregisteredcompanies/repositories/VatRegisteredCompaniesRepository.scala

```diff
@@ -58,39 +69,48 @@ object Wrapper {
 
 @Singleton
 class   VatRegisteredCompaniesRepository @Inject()(
-  reactiveMongoComponent: ReactiveMongoComponent,
+  mongoComponent: MongoComponent,
   bufferRepository: PayloadBufferRepository,
   @Named("deletionThrottleElements") elements: Int,
   @Named("deletionThrottlePer") per: FiniteDuration
 )(implicit val executionContext: ExecutionContext, mat: Materializer) extends
-  ReactiveRepository("vatregisteredcompanies", reactiveMongoComponent.mongoConnector.db, Wrapper.format) {
+  PlayMongoRepository[Wrapper](
+    mongoComponent = mongoComponent,
+    collectionName = "vatregisteredcompanies",
+    domainFormat = Wrapper.format,
+    indexes = Seq(IndexModel(ascending("vatNumber"),
+      IndexOptions().name("vatNumberIndexNew").unique(false).background(true)))) with Logging {
```

Comment:
The index portion in the old code has been commented. Not sure if needed here. Can you please confirm

Reply:
all the code that has been commented out is no longer needed so you can delete it. it was left for your reference.
in this case the old index has been replaced by: " indexes = Seq(IndexModel(ascending("vatNumber"), IndexOptions().name("vatNumberIndexNew").unique(false).background(true))) "

Reply:
commented code removed